src/main/java/net/comments/objects/BCommentTable.java

```diff
@@ -13,8 +13,11 @@ public BCommentTable(WebDriver driver) {
         this.driver = driver;
     }
 
-    public void selectComment(int commentNumber) {
-        this.driver.findElement(By.xpath(String.format("//*[@name = \"SelectedId\" and @value = \"%s\"]", commentNumber + 1))).click();
+    public void selectComment(int[] commentNumbers) {
```

Comment:
If you want to have a method to select several comments, create it. Don't try  to have [GOD methods](https://en.wikipedia.org/wiki/God_object).